rapids_config.cmake

```diff
@@ -25,6 +25,8 @@ else()
       "Could not determine RAPIDS version. Contents of VERSION file:\n${_rapids_version_formatted}")
 endif()
 
+set(rapids-cmake-repo "bdice/rapids-cmake")
+set(rapids-cmake-branch "rapids-cpm-nvtx")
```

Comment:
I got this working locally with help from @KyleFromNVIDIA. We'll need to merge https://github.com/rapidsai/rapids-cmake/pull/606 and revert this change before merging this RMM PR.